src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { FC, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App:FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [visibleTodos, setTodo] = useState(todos);
+  const [isEmptyTitle, isEmptyTitleCheck] = useState(true);
+  const [isUserSelected, isUserSelectedCheck] = useState(true);
+
+  const maxId = Math.max(...visibleTodos.map(todo => todo.id));
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!title) {
+      isEmptyTitleCheck(false);
+
+      return;
+    }
+
+    if (!userId) {
+      isUserSelectedCheck(false);
+
+      return;
+    }
+
+    if (title && userId) {
+      const selectedUser = getUser(userId) || null;
```

Comment:
you already handle it into the function
```suggestion
      const selectedUser = getUser(userId);
```

Reply:
Done!